src/Components/Notifications/util.ts

```diff
@@ -14,3 +17,28 @@ const isArtworksBasedNotification = (
 ) => {
   return ["ARTWORK_ALERT", "ARTWORK_PUBLISHED"].includes(notificationType)
 }
+
+export const setLastSeenNotificationDate = date => {
+  if (!date) {
+    return
+  }
+
+  window.localStorage.setItem(LAST_SEEN_NOTIFICATION_PUBLISHED_AT_KEY, date)
+}
+
+export const hasNewNotifications = lastNotificationDatetime => {
+  if (!lastNotificationDatetime) {
+    return false
+  }
+
+  if (!window.localStorage.getItem(LAST_SEEN_NOTIFICATION_PUBLISHED_AT_KEY)) {
```

Comment:
@dzucconi @damassi what do you think about this approach? will there be any problems with server rendering here? Should we check `isServer`/`typeof window !== "undefined"` here?